web/ui/mantine-ui/src/pages/query/DataTable.tsx

```diff
@@ -34,6 +47,9 @@ export interface DataTableProps {
 }
 
 const DataTable: FC<DataTableProps> = ({ expr, evalTime, retriggerIdx }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
FWIW the CI checks are also failing because of `error    Unused eslint-disable directive (no problems were reported from '@typescript-eslint/no-unused-vars')`

Reply:
Ah yes, while I was working on some stuff I was getting build failures because of unused vars. Forgot to remove this

(even more reason to use older eslint rules)